DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -84,33 +84,28 @@ public EventsSheetsService()
                 new EventResponse(":white_check_mark:", "Yes"),
                 new EventResponse(":grey_question:", "Maybe")
             };
+            var responseList = responses.ToList();
```

Comment:
Do you need this to be a list? 🙂

Reply:
I pass through this list twice to make the two initial rows in the sheet (event response names and emoji). ReSharper warned me there was "multiple enumeration" happening here and recommended I do this. I can switch make to using an `IEnumerable` if that would be better? 🙂

Reply:
Do you need it to be a list in the make method? If not then I'd definitely recommend using `IEnumerable`. Even if you do I'd be tempted to convert to list in the make method instead of here 🙂

Reply:
The multiple enumeration warning happened in the request-making method, but I brought that change up to here. I've now moved the `ToList()` conversion back to the request-making method, so it now takes an `IEnumerable` as an argument 🙂